README.md

```diff
@@ -1,24 +1,132 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <h3><b>HELLO-RAILS-REACT README</b></h3>
+</div>
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [Prerequisites](#prerequisites)
+- [Setup](#setup)
+- [Usage](#usage)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* System dependencies
+# 📖 [HELLO-RAILS-REACT] <a name="about-project"></a>
 
-* Configuration
+**HELLO-RAILS-REACT** is a greeting app that is built rails and react together using webpack and babel as a bundler.
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+    <li>React</li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- **Create the Greeting component that will display a greeting**
+- **Create an API endpoint that selects a random greeting from the database table**
+- **Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting**
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+to install git on your computer. Follow the instruction on the page to do so
+[Click here ](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the instruction on the page to install git.
+[Click here ](https://www.ruby-lang.org/en/documentation/installation/) and follow the instruction on the page to install ruby.
+[Click here ](https://guides.rubyonrails.org/v5.1/getting_started.html) and follow the instruction on the page to install rails.
+[Click here ](https://www.postgresql.org/download/) and follow the instruction on the page to install postgres.
+
+### Setup
+
+Open your preferred folder in the terminal
+
+Clone this repository to your desired folder:
+Using the commands below:
+
+```sh
+  git clone https://github.com/emarkees/hello-rails-react.git
+```
+
+Open the project on your code editor
+
+Change directory to the project folder
+
+```sh
+  cd hello-rails-react
+```
+
+### Usage
```

Comment:
- Please let's make sure we provide all commands necessary to set up the app locally :+1: